### PR TITLE
[dpdk] Emit key name in context file as in bfrt and p4info

### PR DESCRIPTION
**Pull request by kamleshbhalui**

### CODE REVIEW

File: backends/dpdk/dpdkContext.cpp [hunk 1]
```diff
@@ -85,12 +85,13 @@ void DpdkContextGenerator::CollectTablesAndSetAttributes() {
 
 // This functions insert a single key field in the match keys array
 void DpdkContextGenerator::addKeyField(
-Util::JsonArray* keyJson, const cstring name, const IR::KeyElement *key, int position) {
+Util::JsonArray* keyJson, const cstring name, const cstring annon, const IR::KeyElement *key,
```
Comment (mbudiu-vmw) [inline backends/dpdk/dpdkContext.cpp:88]: I would name the parameter "nameAnnotation". annon suggests "annonymous".

**Review by mbudiu-vmw — APPROVED** [review]
None of the other program outputs change?

File: backends/dpdk/dpdkContext.cpp (hunk 1 above)
Comment (kamleshbhalui) [reply backends/dpdk/dpdkContext.cpp]: thank you! done.

**Comment by kamleshbhalui** [comment]
@mbudiu-vmw Please merge it.

**Pull request merged**